deb2rpm.pl

```diff
@@ -102,6 +102,9 @@ sub write_spec {
     if (-e "debian/$package.install") {
       print $out "install_to_buildroot < %{rootdir}/debian/$package.install\n";
     }
+    if (-e "debian/$package.links") {
+      print $out "install_links_in_buildroot < %{rootdir}/debian/$package.links\n";
```

Comment:
Huh? There aren't the diffs that I uploaded! This diff is actually part of commit [42223f29b185f62d5e74bc48ac8764f01dec6c6b](https://github.com/Metaswitch/clearwater-build-infra/commit/42223f29b185f62d5e74bc48ac8764f01dec6c6b).

How odd - I'll try re-uploading the intended diff.